pkg/k8shandler/podtemplate.go

```diff
@@ -79,6 +79,9 @@ func ArePodSpecDifferent(lhs, rhs v1.PodSpec, strictTolerations bool) bool {
 				if different, _ := utils.CompareResources(lContainer.Resources, rContainer.Resources); different {
 					changed = true
 				}
+
+				// we don't need to continue in this inner loop since we found a match
+				break
```

Comment:
since this is the case I think it's probably better to just invert the `if lContainer.Name == rContainer.Name` and then do

```
if lContainer.Name == rContainer.Name {
    continue
}
```

This will reduce nesting and remove the need for the break

Reply:
did you mean this?
```
if lContainer.Name != rContainer.Name {
  contine
}
```

Reply:
doh... yes that's what I meant :laughing:

Reply:
good call, updated